agent/pkg/server/server_init_kubernetes.go

```diff
@@ -120,3 +130,65 @@ func (s *Server) initKubernetesResources() error {
 	}
 	return nil
 }
+
+// podStartWatch log event when pod start began and finished
+func (s *Server) podStartWatch(pods kubeinformersv1.PodInformer, client *kubernetes.Clientset) {
+	pods.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
+		AddFunc: func(obj interface{}) {
+			new := obj.(*v1.Pod)
+			if !strings.HasPrefix(new.Namespace, "modelz-") {
```

Comment:
Shall we reuse the const `ModelzAnnotationValue`?

Reply:
Yes, thanks for remind.